redash/handlers/dashboards.py

```diff
@@ -31,13 +31,12 @@ def get(self):
     @require_permission('create_dashboard')
     def post(self):
         dashboard_properties = request.get_json(force=True)
-        dashboard = models.Dashboard.create(name=dashboard_properties['name'],
-                                            org=self.current_org,
-                                            user=self.current_user,
-                                            layout='[]')
-
-        result = dashboard.to_dict()
-        return result
+        dashboard = models.Dashboard(name=dashboard_properties['name'],
```

Comment:
This should be `models.Dashboard.create`. I'm fixing this in the `webpack` branch, but FYI just in case you plan to use this branch before.

Also the migration sets all dashboards to be drafts.